docs/en/04_Changelogs/4.1.0.md

```diff
@@ -15,25 +9,114 @@ guide developers in preparing existing 3.x code for compatibility with 4.0
 
 ### Upgrade `public/` folder
 
-By default all new projects installed from 4.1.0 will have a `public/` webroot folder. This folder name is
-not configurable, but is turned on by creating this file, and off by ensuring this folder doesn't exist.
+By default all new projects installed from 4.1.0 will have a `public/` webroot folder which separates
+all web-accessible files from protected project assets, providing an additional level of security.
+
+This folder name is not configurable, but is turned on by creating this folder, and off by ensuring
+this folder doesn't exist.
 
 When separating the public webroot from the BASE_PATH it is necessary to move a few files during migration:
 
  - Move `.htaccess` from base to `public/`
  - Move `index.php` from base to `public/`
- - Move `assets` folder (including the nested `assets/.protected` folder) into `public/`
+ - Move `assets` folder (including the nested `assets/.protected` folder) into `public/`.
+   This is the only folder which needs write permissions.
  - Ensure that the `public/resources` folder exists; If this folder already exists in root, you should
    delete this, and re-generate it by running `composer vendor-expose` in your root path.
  - Any public assets committed directly to your project intended to be served directly to the
  webserver. E.g. move `mysite/javascript/script.js` to `public/javascript/script.js`.
+ You can then use `Requirements::css('javascript/script.js');` /
+ `<% require css('javascript/script.js') %>` to include this file.
  - Ensure that the web-root configured for your server of choice points to the public/ folder instead of the base path.
- 
+  E.g. an apache virtualhost configuration would look like:
+
+```
+<VirtualHost *:80>
+  ServerName mywebsite.com
+  ServerAlias *.mywebsite.com
+  VirtualDocumentRoot "/var/www/Sites/mywebsite/public/"
+</VirtualHost>
+```
+
 You may also need to add various changes to your code if you reference the BASE_PATH directly:
 
  - You should use `Director::publicFolder()` instead of `Director::baseFolder()` if referring to the
  public folder.
  - You can check if a public folder exists with `Director::publicDir()`
+
+### Example `public/` folder structure
+ 
+For example, this is an existing folder structure:
+
+```
+- assets/
```

Comment:
Marking this up as a nested list in pseudo-Markdown seems a bit weird - I don't think we do that in other places around the docs. I find it harder to visually parse this way.

More importantly: Naming multiple files the same (`style.css`), where only *some* are symlinks of others is very confusing.